source/content.ts

```diff
@@ -88,4 +88,12 @@ import './features/bypass-checks';
 import './features/add-co-authored-by';
 
 // Add global for easier debugging
+type Select = typeof select;
+
+declare global {
+	interface Window {
+		select: Select;
```

Comment:
Can this be `select: typeof select`?

Reply:
I had it like that initially and then then linter was complaining even though `select` is imported in that file. Let me see if I can do some magic.

![image](https://user-images.githubusercontent.com/833231/52191062-bb642100-2810-11e9-8166-e1677daad47d.png)

Reply:
This should be fine by just grabbing the `SelectDom` interface instead from select-dom. I was being too fancy.